tools/BuildAssets/BuildTasks/Microsoft.Azure.Sdk.Build.Tasks/BootstrapTasks/GetBuildTools.cs

```diff
@@ -190,11 +190,33 @@ public override bool Execute()
             finally
             {
                 WebCopier?.Dispose();
+                VerifyCopiedFiles();
```

Comment:
@dsgouda can you check what unableToCopyFilePath is meant to do.
Seems like we might be doing some logging if we are unable to copy files.
Also if have to call VerifyCopiedFiles, let's call it inside try prior to report errors. But first check why are we initializing the list unableToCopyFilePath